### ISSUE TITLE
Moment.js: Maximum call stack size exceeded

### ISSUE THREAD

**Issue by JulianBerger** [michaeldegroot/socket-anti-spam#13]
I get a Maximum call stack size exceeded error from moment.js on start.

I'm using Node.js v6.8.1 , socket.io v1.5 and socket-anti-spam v1.1.6-c 

```
/home/nginx/domains/domain.com/nodejs/node_modules/moment/moment.js:80
    function defaultParsingFlags() {
                                ^

RangeError: Maximum call stack size exceeded
    at defaultParsingFlags (/home/nginx/domains/domain.com/nodejs/node_modules/moment/moment.js:80:33)
    at getParsingFlags (/home/nginx/domains/domain.com/nodejs/node_modules/moment/moment.js:100:21)
    at valid__isValid (/home/nginx/domains/domain.com/nodejs/node_modules/moment/moment.js:125:25)
    at prepareConfig (/home/nginx/domains/domain.com/nodejs/node_modules/moment/moment.js:2361:14)
    at createFromConfig (/home/nginx/domains/domain.com/nodejs/node_modules/moment/moment.js:2325:44)
    at createLocalOrUTC (/home/nginx/domains/domain.com/nodejs/node_modules/moment/moment.js:2412:16)
    at local__createLocal (/home/nginx/domains/domain.com/nodejs/node_modules/moment/moment.js:2416:16)
    at utils_hooks__hooks (/home/nginx/domains/domain.com/nodejs/node_modules/moment/moment.js:16:29)
    at authenticate (/home/nginx/domains/domain.com/nodejs/node_modules/socket-anti-spam/antispam.js:144:31)
    at Object.exports.addSpam (/home/nginx/domains/domain.com/nodejs/node_modules/socket-anti-spam/antispam.js:58:3)
```

**Comment by michaeldegroot** [comment]
Hi, thanks for reporting, seems travis picked this up as well, If I have time for it I will look at it :)

http://i.imgur.com/ZXwqp0m.png

**Comment by michaeldegroot** [comment]
Just confirmed this locally with a quick look when upgrading socket io from 1.3.7 to 1.5.0

I will look at this probably this weekend!

**Comment by michaeldegroot** [comment]
Fixed and published on NPM as socket-anti-spam@1.1.8

Thanks for your report
